haystack/pipelines/standard_pipelines.py

```diff
@@ -159,6 +165,10 @@ def eval(
         params: Optional[dict] = None,
         sas_model_name_or_path: Optional[str] = None,
         add_isolated_node_eval: bool = False,
+        custom_document_id_field: Optional[str] = None,
+        context_matching_min_length: int = 100,
+        context_matching_boost_split_overlaps: bool = True,
+        context_matching_threshold: float = 65.0,
```

Comment:
Any intuition behind that threshold that we can write down in the doc string? And now that I am thinking about it, as it is a float and not an int, would it be more intuitive to have a score between 0 and 1?

Reply:
Yep, from an empirical perspective 65 seems to be a good value if you have partially overlapping contexts. Checkout https://github.com/deepset-ai/haystack/blob/c7d988c23ae5e6bc7cc86b83c3ced1141d13a873/test/test_utils.py#L181-L194 and https://github.com/deepset-ai/haystack/blob/c7d988c23ae5e6bc7cc86b83c3ced1141d13a873/test/test_utils.py#L150-L167. Even with noise this threshold produces above 99% accuracy. However I couldn't make a large scale study on that yet. And of course there might be cases like lots of same structured tables with different contents where you might want to increase that value further. We can add that however as a rule of thumb to the docstrings.

Regarding the value range: yes I also don't like it here. The range between 0.0 and 100.0 comes basically from the underlying rapidfuzz (fuzzy-wuzzy successor). I would fix this in another PR.